catalog/Testing/continuous_testing.yml

```diff
@@ -9,6 +9,8 @@ projects:
   - guard-rubocop
   - infinity_test
   - kicker
+  - minitest-flash
+  - minitest-substitute
```

Comment:
@svoop We already have minitest-substitute in the mocking category, and it seems more fitting there, I think we should just put minitest-flash into this category, WDYT?

Reply:
Oh, my bad @colszowka, I was in such a flow. :smile: I'll force push it out of this PR.

Reply:
There you go, the dupe is removed.